receiver/hostmetricsreceiver/internal/scraper/networkscraper/network_scraper_test.go

```diff
@@ -155,8 +170,7 @@ func TestScrape(t *testing.T) {
 	}
 }
 
-func assertNetworkIOMetricValid(t *testing.T, metric pdata.Metric, descriptor pdata.Metric, startTime pdata.Timestamp) {
-	internal.AssertDescriptorEqual(t, descriptor, metric)
```

Comment:
I think we should we pass metric names to this function and validate it instead of the old descriptor

Reply:
Thanks. Let me take a look.

Reply:
Fixed. Added assert to check whether metric.Name() matches.